.github/CODEOWNERS

```
@@ -0,0 +1,33 @@
+# Core
+/airflow/executors/ @kaxil @XD-DENG @ashb @turbaszek
+/airflow/jobs/ @kaxil @ashb @XD-DENG
+/airflow/models @kaxil @XD-DENG @ashb @turbaszek
+
+# DAG Serialization
+/airflow/serialization @kaxil
```

```suggestion
/airflow/serialization/ @kaxil
```

To keep consistent with other lines and the [sample syntax](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax)